libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt

```diff
@@ -67,6 +68,7 @@ class RouteProgress internal constructor(
     val stale: Boolean,
     val routeAlternativeId: String?,
     val currentRouteGeometryIndex: Int,
+    val alternativeRoutesIndices: Map<String, RouteIndices>,
```

Comment:
maybe expose it as an experimental? 
There is always a chance that we decide to get rid of it or change later

Reply:
TBH, I still feel that it's better not to expose it even as an experimental until explicit customer request with justification, just to let us easier change SDK in the future.